action.yaml

```diff
@@ -308,10 +308,19 @@ runs:
         branch=$(echo "${GITHUB_REF#refs/heads/}")
         echo "Branch pushed to: $branch"
         git fetch origin $branch
-        files=$(git diff --name-only ${{ github.event.before }} ${{ github.event.after }})
-        echo "Files changed: $files"
+
         DBT_DEPLOY=false
 
+        # case when the triggered event is a new tag creation, then we would need to deploy the dbt project to be on the safe side
```

Comment:
nit: "on the safe side" is a bit vague. We should say that we have to deploy the dbt project (or image below) because we cannot determine that it does not need to be deployed